.storybook/themes/news.scss

```diff
@@ -466,6 +465,63 @@
 				),
 			),
 			blocks: (
+				simple-list: (
+					gap: var(--global-spacing-4),
+					components: (
+						stack: (
+							min-height: 72px,
+						),
+						image: (
+							vertical-align: middle,
+						),
+						link: (
+							text-decoration: none,
+						),
+					),
+				),
+				simple-list-title: (
+					font-size: var(--global-font-size-7),
+				),
+				simple-list-item: (
+					gap: var(--global-spacing-6),
+				),
+				simple-list-item-anchor: (
+					flex: 0 0 33%,
+				),
+				simple-list-headline-text: (
+					-webkit-line-clamp: 3,
```

Comment:
```suggestion
```

can be removed if you add the props `truncationLines` to the Heading

Reply:
powerful info

Reply:
looks like this handles the ellipsis as well so this whole class can go

Reply:
woo!